feature/schedule/src/main/java/see/day/schedule/component/bottomsheet/AlertTimeBottomSheet.kt

```
@@ -221,17 +313,43 @@ private fun AlertTimeBottomSheetPreview() {
+fun formatTimeToKorean(hour: Int, minute: Int = 0): String {
+    require(hour in 0..23) { "Hour must be between 0 and 23" }
+    require(minute in 0..59) { "Minute must be between 0 and 59" }
+
+    val period = if (hour < 12) "오전" else "오후"
+    val displayHour = when (hour) {
+        0 -> 12
+        in 1..12 -> hour
+        else -> hour - 12
+    }
+
+    return String.format("%s %02d:%02d", period, displayHour, minute)
```

detekt (1.23.8)

[warning] String.format("%s %02d:%02d", period, displayHour, minute) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)